zcbor/zcbor.py

```diff
@@ -2269,12 +2269,12 @@ def range_checks(self, access):
 
     # Return the full code needed to encode/decode this element, including children,
     # key and cbor, excluding repetitions.
-    def repeated_xcode(self, union_uint=None):
+    def repeated_xcode(self, union_int=None):
         range_checks = self.range_checks(self.val_access())
         xcoder = {
             "INT": self.xcode_single_func_prim,
-            "UINT": lambda: self.xcode_single_func_prim(union_uint),
-            "NINT": self.xcode_single_func_prim,
+            "UINT": lambda: self.xcode_single_func_prim(union_int),
+            "NINT": lambda: self.xcode_single_func_prim(union_int),
```

Comment:
Why is it now possible to pass this as an argument? Was the old behavior buggy, or is this more than a rename?

Reply:
The optimization was expanded to all ints, not only uints. This is essentially where that change happened.